var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -45,6 +46,7 @@ class Vtk(CMakePackage):
     # VTK 8.1, that should change
     conflicts('+osmesa', when='+qt')
     conflicts('^python@3:', when='@:8.0')
+    conflicts('^python@3.8:', when='@:8.9 +python')
```

Comment:
These Python conflicts should be converted to dependency statements. `conflicts` doesn't influence the concretizer, it just causes it to crash. `depends_on` actually influences the decision the concretizer makes.